catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/LibraryImpl.java

```diff
@@ -254,7 +254,7 @@ public LibraryManager get_manager(String manager_type, AccessCriteria access_cri
                     poa.activate_object_with_id(managerId.getBytes(Charset.forName(NsiliEndpoint.ENCODING)),
                             standingQueryMgr);
                 } catch (ServantAlreadyActive | ObjectAlreadyActive | WrongPolicy e) {
-                    LOGGER.error("Error activating StandingQueryMgr: {}", e);
+                    LOGGER.info("Error activating StandingQueryMgr: ", e);
```

Comment:
I noticed here we set this to info, yet in other places with a similar message about activation we set it to debug. Can we set these to all info or all debug for consistency?